app/views/tournament/side.scala

```diff
@@ -60,7 +60,9 @@ object side:
             markdownLinksOrRichText(d)
           ),
         tour.looksLikePrize option bits.userPrizeDisclaimer(tour.createdBy),
-        views.html.gathering.verdicts(verdicts, tour.perfType),
+        tour.isFinished option views.html.gathering.verdicts(verdicts, tour.perfType, false) orElse Some(
+          views.html.gathering.verdicts(verdicts, tour.perfType)
```

Comment:
Would it be possible to pass `!tour.isFinished` as the 3rd param?

Reply:
That is actually a good idea! fixed it in the last commits.